core/src/main/java/org/jboss/modcluster/mcmp/impl/DefaultMCMPRequestFactory.java

```diff
@@ -63,7 +63,7 @@ public MCMPRequest createConfigRequest(Engine engine, NodeConfiguration nodeConf
         // If address was specified as a host name, we would prefer it
         // toString() will not perform reverse dns lookup
         // so send host name portion, if it exists
-        String address = connector.getAddress().toString();
+        String address = connector.getAddress().toString().toLowerCase();
```

Comment:
I had that in my first version. But couldn't think of an address where it would make a difference.